### PR TITLE
Arena

### PR DESCRIPTION
**Pull request by IamVelcroboy**
Changelog:

Map Changes:
-Moves RobVend to drone lab in Engineering
-Moves Circuit Imprinter to Engineering
-Removes ATVs from Engineering
-Adds Windoors in Kitchen to prevent griefers
-Redone "Secrets" to "be best."
-Replaces wt5500 with Vectors
-Replaces Tritium miner with NO2 Miner 

Yaml Edits:
-Updates Xeno spawner for NPC only
-Adds Rng Variety to Carp Spawner
-Removes duplicate file in wrong location

### CODE REVIEW

**Comment by Elijahrane** [comment]
it'd be easier to remove the takeover from xenos and then make a file where it's just children of them with takeover available rather than copying + pasting their dozens of comps

**Comment by IamVelcroboy** [comment]
ok

**Comment by IamVelcroboy** [comment]
Yeah, I don't know what I did with the Tortuga update, so when this is merged, I'm going to reset and push again

File: Resources/Prototypes/Entities/Mobs/NPCs/xeno.yml [hunk 1]
```diff
@@ -431,3 +466,234 @@
         Base: dead_small_purple_snake
   - type: SolutionTransfer
     maxTransferAmount: 1
+
+# NPC versions
```
Comment (Elijahrane) [inline Resources/Prototypes/Entities/Mobs/NPCs/xeno.yml]: NPC versions should be the parent and non-NPC should be the child

Comment (IamVelcroboy) [reply Resources/Prototypes/Entities/Mobs/NPCs/xeno.yml]: That is how it is, there's just the one parent that they all inherit from...but I see I need a non-npc version of it.

Comment (Elijahrane) [reply Resources/Prototypes/Entities/Mobs/NPCs/xeno.yml]: the issue is that MobXenoPraetorianNPC and MobXenoPraetorian are parenting the same base and duplicating all of their components

It should go MobXeno -> MobXenoPraetorian -> MobXenoPraetorianGhost which does not duplicate any components and simply adds the ghost takeover

Comment (IamVelcroboy) [reply Resources/Prototypes/Entities/Mobs/NPCs/xeno.yml]: OK...I'll reconfigure it that way

Comment (IamVelcroboy) [reply Resources/Prototypes/Entities/Mobs/NPCs/xeno.yml]: nm, that last commit then

**Comment by Elijahrane** [comment]
let me rerun tests rq i broke them

**Comment by Elijahrane** [comment]
uhhh the yaml is broken

**Comment by IamVelcroboy** [comment]
hmm, where?

**Comment by Elijahrane** [comment]
read the error and run yaml linter locally

**Pull request merged**